### DISCUSSION TITLE
space on the right (page and post)

### DISCUSSION THREAD

**Discussion by jphconstantin** [mmistakes/minimal-mistakes#2567] [Q&A] [answered]
Hello,
Instead of the toc, what can be displayed on the right ?
Is there the possibility to extend the page/post width when no toc are displayed ?
rgds,

**Comment by iBug** [comment, marked as answer]
<https://mmistakes.github.io/minimal-mistakes/docs/layouts/#wide-page-1>

**Comment by jphconstantin** [comment]
Thanks !
Sorry for my crazy question. I am a newbie with Jekyll, github, and this fabulous theme !

and what about my first question: toc or what else ?

**Comment by iBug** [comment]
By default, only ToC or extended post content (with wide layout). If you want to display something else on the right side, you'll have to dig into the theme and make changes on your own.

**Comment by jphconstantin** [comment]
I am discovering the theme. Actually I try to know it better.

The home page layout display the name of the posts.
Is it possible to display more: e.g the first N lines of a post and a button "read more" ?
